### PR TITLE
docs: add link to rehype remove images

### PR DESCRIPTION
**Pull request by iloveitaly**
### Initial checklist

* [x] I read the support docs 
* [x] I read the contributing guide 
* [x] I agree to follow the code of conduct 
* [x] I searched issues and couldn’t find anything (or linked relevant results below) 
* [x] If applicable, I’ve added docs and tests

### Description of changes
adding link to new plugin

ref: https://github.com/rehypejs/rehype-minify/pull/47

### CODE REVIEW

**Review by wooorm — COMMENTED** [review]
- I do still recommend using `unist-util-visit`, much more powerful! https://github.com/rehypejs/rehype-minify/pull/47#pullrequestreview-1525875930
- you don’t have to return `undefined` explicitly I guess? Don’t return anything is the same?
- Looks like `index.d.ts` is generated, so perhaps you can omit it from the Git history

**Comment by iloveitaly** [comment]
> * I do still recommend using `unist-util-visit`, much more powerful

for me, it felt like this method was a bit more explicit / easier to understand. It's such a simple use case I figured it didn't matter much. Will look into this for the next one though!

File: doc/plugins.md [hunk 1]
```diff
@@ -144,6 +144,7 @@ The list of plugins:
     — compile to React
 *   [`rehype-remark`](https://github.com/rehypejs/rehype-remark)
     — remark support
+*   [`rehype-remove-images`](https://github.com/iloveitaly/rehype-remove-images)
```
[the remark this answers is not in the document]
Comment (iloveitaly) [reply doc/plugins.md]: Done!

File: doc/plugins.md [hunk 2]
```diff
@@ -8,10 +8,12 @@ This page lists existing plugins.
...
-*   [List of plugins](#list-of-plugins)
-*   [List of utilities](#list-of-utilities)
-*   [Use plugins](#use-plugins)
-*   [Create plugins](#create-plugins)
+- [Plugins](#plugins)
+  - [Contents](#contents)
+  - [List of plugins](#list-of-plugins)
+  - [List of utilities](#list-of-utilities)
+  - [Use plugins](#use-plugins)
+  - [Create plugins](#create-plugins)
```
Comment (wooorm) [inline doc/plugins.md:16]: Can you run `npm install` and `npm test`, that should reformat everything according to project style, or undo these changes manually? Thanks!

Comment (iloveitaly) [reply doc/plugins.md]: done!

**Review by wooorm — APPROVED** [review]

**Comment by wooorm** [comment]
Thanks!
